front/temporal/labs/utils/gong.ts

```diff
@@ -80,27 +97,88 @@ export async function retrieveGongTranscripts(
 }
 
 export async function retrieveGongTranscriptContent(
+  auth: Authenticator,
```

Comment:
`nit` No action required.

I'd suggest that you try to split and structure your code more effectively. Using named functions with clear, descriptive names can help keep the code clean and make it significantly easier to review 🙃.